formfactory/actions.py

```diff
@@ -98,3 +98,9 @@ def login(form_instance, **kwargs):
     )
     if user is not None:
         auth.login(request, user)
+
+
+@register
+def wizard_done_step(form_dict, **kwargs):
```

Comment:
I was using this to experiment with what should happen at the end of a wizard. Removing it now.